datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemEditTextIntegerViewHolderFactory.kt

```diff
@@ -38,14 +43,19 @@ internal object QuestionnaireItemEditTextIntegerViewHolderFactory :
         questionnaireItemViewItem: QuestionnaireItemViewItem
       ) {
         val input = editable.toString()
+        if (input.isNullOrEmpty()) {
```

Comment:
Since `editable` is non-nullable, do we need the `isNull`?
```suggestion
        if (input.isEmpty()) {
```